script/Ciel/Day 7/07_01_CIEL07_9.txt

```diff
@@ -781,123 +843,136 @@ It's quite ironic.
 [sha:ff9084d428cea6b7187b8617eaa9583db7c21410]{
 -- Page 694, Offset 16541.
 -- 無限に広がる人間を恐怖した男が、その解決として、無限の時間を求めたのだ。
-But the man who had so feared the infinitely expanding humanity, found his solution in the pursuit of infinite time.
+The man, who so feared the infinite expansion of humanity,
+ found his solution in the pursuit of infinite time.
 }
 [sha:b4c3e764230baee58c336b47e98be4ac8278fa10]{
 -- Page 695, Offset 16542.
 -- 『……ふうん。まあ、ここらで足抜けするのも悪くない判断だよ。例の改革はともかく、<印刷技術|グーテンベルグ>の一般化が致命的だからな。しばらく荒れるだろ、<教|う><会|ち>も協会も。
-// lint-off:LintPageOverflow
-// possibly the longest page in the game, needs desperate help
-"Hmm... well, breaking your ties with us now isn't a bad call. The rapid spread of the Gutenberg press has proven critical, not to mention the whole reformation thing. It's gonna get rough for a while, both for us of the Church, and the Association.
+// Unvoiced
+// Possibly the longest page in the game, edit at your peril
+// Ruby used to cram more shit in, forgive me
+'Hmm... well, breaking ties with us now isn't a bad call.
+ The rapid spread of the Gutenberg press is a real problem,
+ to say nothing of the reformation.
+ Gonna be a rough time, both for <us|T h e   C h u r c h   > and the Association.
 }
 [sha:d13105c554ffd15c0d1b754253e1cef65c34206f]{
 -- Page 695, Offset 16543.
 -- もっとも<魔術協会|あっち>の方はより大打撃だろうがね。大量印刷によってあらゆる迷信が駆逐される。神秘の屋台骨は派手に軋むだろうさ。
-Though of course, I suspect the Mages Association is going to take the heavier blow.
- Most superstitions will be eradicated thanks to mass printing――the foundational structure of the arcane will surely groan under the stress.
+I imagine <they'll|T h e  M a g e s   A s s o c i a t i o n   > bear the brunt of it though. A bunch
+ of superstitions dying in the wake of mass printing's
+ gonna seriously stress the foundations of the arcane.
 }
 [sha:de4a211a399d8dd48c6efd68377d860b2dafd2b0]{
 -- Page 695, Offset 16544.
 -- ああ、だから今なのか。どちらも庭掃除で手一杯。トチ狂った神童が真祖どもの城を目指したところですぐに追っ手は差し向けられん。まさに火事場泥棒だ。この隙に一番手近な永遠を手に入れようとするんだね、君は』
-// there is perhaps a better way of phrasing まさに火事場泥棒だ in English (currently: Talk about taking advantage of a bad situation)
-Oh, I suppose that's why you'd go with this timing, since both of them will be so preoccupied with their own internal affairs.
- Even if some mad wonder child was to set out for the True Ancestor's castle, they wouldn't be able to send someone after them immediately.
- Talk about taking advantage of a bad situation. You're gonna use this opportunity to acquire the kind of eternity closest at hand, aren't you?"
+Guess that explains you heading for the True Ancestor's castle now,
+ while they've both got their hands too full to give chase.
```

Comment:
Man, I know it's 仕方がない and all, but this page just missing whole sentences because of page limit breaks my heart.
If there ever is some coding that solves monospacing this is the place we need to come back to.

Reply:
Yeah, I had to do some serious snipping to cut 18 in game lines down to 12. 
We could also try messing with the scripting to break it into multiple pages. Tinkering too deeply tends to break things in mysterious ways, though mostly around choices, and this file doesn't end in one.

Reply:
Alright, I shoved the final stanza on a new page and reinflated the text a bit.